test/Table_Tests/src/In_Memory/Text_Cleanse_Spec.enso

```diff
@@ -0,0 +1,189 @@
+            res3 = test_col_with_nums_and_letters.text_cleanse [Text_Cleanse.Letters, Text_Cleanse.Numbers]
+            res1.should_equal (Column.from_vector "Test" ["ABC", "DEF", "GHI", "JKL"])
+            res2.should_equal (Column.from_vector "Test" ["1234", "5678", "9012", "3456"])
+            res3.should_equal (Column.from_vector "Test" ["", "", "", ""])
+        group_builder.specify "should remove punctuation" <|
+            test_col_with_punctuation = Column.from_vector "Test" ['Hello, World!', 'How are you?', ',.!?():;\'"']
+            res = test_col_with_punctuation.text_cleanse [Text_Cleanse.Punctuation]
+            res.should_equal (Column.from_vector "Test" ["Hello World", "How are you", ""])
+        group_builder.specify "should remove symbols" <|
+            test_col_with_symbols = Column.from_vector "Test" ['Hello, World123!', 'How_are_you?', ',.!?():;\'"', '🥇🕒🕒']
+            res = test_col_with_symbols.text_cleanse [Text_Cleanse.Symbols]
+            res.should_equal (Column.from_vector "Test" ["Hello World123", "Howareyou", "", ""])
+        group_builder.specify "should error if applied to non-text column" <|
+            test_col_num = Column.from_vector "Test" [1, 2, 3, 4]
+            test_col_num.text_cleanse [Text_Cleanse.Numbers] . should_fail_with Invalid_Value_Type
+        group_builder.specify "should apply the operations in order" <|
+            test_col_with_mixed_chars = Column.from_vector "Test" ["  11String with Leading Spaces then Leading Numbers", "22  String with Leading Numbers then Leading Spaces"]
+            res1 = test_col_with_mixed_chars.text_cleanse [Text_Cleanse.Leading_Whitespace, Text_Cleanse.Leading_Numbers]
+            res2 = test_col_with_mixed_chars.text_cleanse [Text_Cleanse.Leading_Numbers, Text_Cleanse.Leading_Whitespace]
+            res3 = test_col_with_mixed_chars.text_cleanse [Text_Cleanse.Leading_Numbers, Text_Cleanse.Leading_Whitespace, Text_Cleanse.Leading_Numbers]
+            expected_col1 = Column.from_vector "Test" ["String with Leading Spaces then Leading Numbers", "  String with Leading Numbers then Leading Spaces"]
+            expected_col2 = Column.from_vector "Test" ["11String with Leading Spaces then Leading Numbers", "String with Leading Numbers then Leading Spaces"]
+            expected_col3 = Column.from_vector "Test" ["String with Leading Spaces then Leading Numbers", "String with Leading Numbers then Leading Spaces"]
+            res1.should_equal expected_col1
+            res2.should_equal expected_col2
+            res3.should_equal expected_col3
+    suite_builder.group "Text Cleanse" group_builder->
+        input = "  It was the best  of  times  it was  the  worst  of  times  "
+        group_builder.specify "should remove leading whitespace" <|
+            expected = "It was the best  of  times  it was  the  worst  of  times  "
+            res = input.cleanse [Text_Cleanse.Leading_Whitespace]
+            res.should_equal expected
```

Comment:
It feels like this group belongs in `Base_Tests`, not here.

Reply:
Done